### PR TITLE
GetCredit info for a product

### PR DESCRIPTION
**Pull request by wnowicki**
## Features
- Get Credit Info for a Product

### CODE REVIEW

**Comment by EvanBarbour3** [comment]
@wnowicki

**Comment by EvanBarbour3** [comment]
@00eadeyemi

File: src/Gateways/ProductGateway.php [hunk 1]
```diff
@@ -42,4 +42,22 @@ public function getProductGroupsWithProducts($extId, $token)
...
+     * @param array $params
+     * @return array
+     */
+    public function getCreditInfo($installation, $product, $token, array $params)
+    {
+        return $this->postDocument(
+            '/v4/installations/' . $installation . '/products/' . $product . '/get-credit-information',
+            $params,
+            $token,
+            'CreditInfo'
```
Comment (00eadeyemi) [inline src/Gateways/ProductGateway.php]: CS

**Comment by 00eadeyemi** [comment]
Looks good I could not find anything wrong.

**Pull request merged**